### ISSUE TITLE
Stuck Train images  in RTX 4090 

### ISSUE THREAD

**Issue by MarcosEngesult** [graphdeco-inria/gaussian-splatting#319]
after the comand python train.py -s

(gaussian_splatting) C:\Users\marcos.costa\gaussian-splatting>python train.py -s data/gerdau -r 1
Optimizing
C:\Users\marcos.costa\AppData\Local\anaconda3\envs\gaussian_splatting\lib\site-packages\torch\cuda\__init__.py:106: UserWarning:
NVIDIA GeForce RTX 4090 with CUDA capability sm_89 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 compute_37.
If you want to use the NVIDIA GeForce RTX 4090 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(incompatible_device_warn.format(device_name, capability, " ".join(arch_list), device_name))
Output folder: ./output/8f6b6f3b-8 [13/10 20:33:05]
Tensorboard not available: not logging progress [13/10 20:33:05]
Reading camera 149/149 [13/10 20:33:06]
Loading Training Cameras [13/10 20:33:06]

and nothing happen after that.

**Comment by skymanaditya1** [comment]
I have the same GPU and had the issues. I fixed it by setting up CudaToolkit version 11.8 through -- 

conda install -c "nvidia/label/cuda-11.8.0" cuda-toolkit

And then setup pytorch version 2.1 and corresponding compatible torchvision and torchaudio versions. Torchvision ==0.16.0+cu118 and torch==2.1.0+cu118, torchaudio==2.1.0+cu118. 

The way you install this is through -- pip3 install torch==2.1 torchvision==0.16 torchaudio==2.1 --index-url https://download.pytorch.org/whl/cu118

After this succeeds, install the two submodules required via pip, and pip install any of the remaining dependencies. It should work after that.
